projects/dark-matter-reana.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: REANA workflow for Dark Matter Searches
+postdate: 2023-02-14
+categories:
+  - Analysis tools
+durations:
+  - 3 months
+skillset:
+  - Python
+  - Docker
+status:
+  - Available
+project:
+  - IRIS-HEP
+experiments:
+  - Any
+program:
+  - Any
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Implement a REANA workflow for dark matter searches at RCFM
+description: >
+  [REANA](https://reana.io/) is a platform for reproducible data analysis workflows
+  that can be run at scale. REANA has been used extensively for running containerized
+  workflows of LHC experiments, like ATLAS, and for reinterpretation of published
+  analyses. This project would aim to implement a REANA workflow for a dark matter
+  search at RCFM to improve replicability of analyses and to provide a starting
+  point for future RCFM analyses.
```

Comment:
Still need to revise this to make the role and description of RCFM clear. Then we can rebase and review this.